ext/fg/js/popup-proxy.js

```diff
@@ -21,14 +21,14 @@
  */
 
 class PopupProxy {
-    constructor(id, depth, parentId, parentFrameId, url) {
+    constructor(id, depth, parentId, parentFrameId, url, applyFrameOffset=async (x, y) => [x, y]) {
```

Comment:
Default `applyFrameOffset` to `null` instead, and then do a `null` check before invoking it later on. This simplifies the default arguments (looks complex) and can also elide an `async` call later on when null.